.env.sample

```diff
@@ -1,6 +1,7 @@
 BITCOIND_URL=
 EMILY_URL=
 MEMPOOL_API_URL=https://leather.mempool.space/api
+MEMPOOL_URL=https://leather.mempool.space
```

Comment:
We can't use these endpoints and they shouldn't be in the sample. Just refer to the standard mempool URLs (i.e. drop the `leather` prefix).